docs/src/use/configure/migration-guide.md

```diff
@@ -20,9 +23,11 @@ For reference information on these configuration formats, refer to the following
 
 To get started, use the [configuration migrator](https://npmjs.com/package/@eslint/migrate-config) on your existing configuration file (`.eslintrc`, `.eslintrc.json`, `.eslintrc.yml`), like this:
 
-```shell
-npx @eslint/migrate-config .eslintrc.json
-```
+{{ npx_tabs({
+    package: "eslint",
+    args: ["@eslint/migrate-config", ".eslintrc.json"],
+    comment: null
+}) }}
```

Comment:
```suggestion
{{ npx_tabs({
    package: null,
    args: ["@eslint/migrate-config", ".eslintrc.json"],
    comment: null
}) }}
```

this can be null so command will be
```
npx @eslint/migrate-config .eslintrc.json
```